### DISCUSSION TITLE
Alpine Package Update Expectation (Nextcloud)

### DISCUSSION THREAD

**Discussion by MrDevanWright** [tteck/Proxmox#3977] [Q&A] [answered]
[Per a previous post](https://github.com/tteck/Proxmox/discussions/3722), we're advised to update [Nextcloud via the Alpine Package Keeper](https://pkgs.alpinelinux.org/package/v3.20/community/x86_64/nextcloud), `apk update && apk upgrade`. As [Nextcloud is now onto version 30](https://nextcloud.com/changelog/#latest30), **what is the expected timeline for APK updates?** Understandably, it only came out a week ago so it will take some time for the APK to be updated. I've never utilized this platform before so I'd like to learn a realistic expectation.

Additionally, why Alpine Linux?

**Comment by tteck** [comment, marked as answer]
>what is the expected timeline for APK updates?

You never know when/if the package maintainer will release an upgrade.

>Additionally, why Alpine Linux?

Only you can answer that, being you selected Alpine over Debian.

**Reply by MrDevanWright** [reply]
Per the [helper-script](https://tteck.github.io/Proxmox/#nextcloud-lxc), Nextcloud Hub only installs on Alpine? When I go through the installer it only offers Alpine as an option. Is there a helper-script with other options and all the Nextcloud features?

**Reply by tteck** [reply]
They're all the same.
